### PR TITLE
New Menu Items for disabling Info & Search buttons in the menu

### PR DESCRIPTION
**Pull request by tr12345**
Add menu options for disabling Info and Search buttons.  Wanted a way to disable these buttons as I don't use these features and they conflict with other mappings.  For example using Select+Start to enter and exit the menu.  Once you've entered the menu once the next time you go into the menu the info screen is displayed.

## Guidelines

1. Rebase before opening a pull request
2. If you are sending several unrelated fixes or features, use a branch and a separate pull request for each
3. If possible try squashing everything in a single commit. This is particularly beneficial in the case of feature merges since it allows easy bisecting when a problem arises

## Description

[Description of the pull request, detail any issues you are fixing or any features you are implementing]

## Related Issues

[Any issues this pull request may be addressing]

## Related Pull Requests

[Any other PRs from related repositories that might be needed for this pull request to work]

## Reviewers

[If possible @mention all the people that should review your pull request]

### CODE REVIEW

File: intl/msg_hash_us.h [hunk 1]
```diff
@@ -2612,6 +2612,22 @@ MSG_HASH(
    MENU_ENUM_SUBLABEL_INPUT_UNIFIED_MENU_CONTROLS,
    "Use the same controls for both the menu and the game. Applies to the keyboard."
    )
+MSG_HASH(
+   MENU_ENUM_LABEL_VALUE_INPUT_DISABLE_INFO_BUTTON,
+   "Disable Info Button"
+   )
+MSG_HASH(
+   MENU_ENUM_SUBLABEL_INPUT_DISABLE_INFO_BUTTON,
+   "If enabled Info button presses will be ignored"
```
Comment (sonninnos) [inline intl/msg_hash_us.h:2621]: Please end these sublabels with a dot.

**Comment by hizzlekizzle** [comment]
This functionality has been requested >1mil times. Huge thanks for the contribution!

**Comment by Cthulhu-throwaway** [comment]
Maybe the advanced flag should be set for those in order to prevent users from enabling them by mistake and getting confused over those buttons not working.

File: intl/msg_hash_us.h (hunk 1 above)
Comment (tr12345) [reply intl/msg_hash_us.h]: Updated

**Comment by tr12345** [comment]
> Maybe the advanced flag should be set for those in order to prevent users from enabling them by mistake and getting confused over those buttons not working.

My personal preference would be for them to not have the advanced flag.  I think it's easy enough to toggle back off if someone turns them on by accident.  Plus the setting is immediate and does not require restart to go into effect.

**Comment by Cthulhu-throwaway** [comment]
I for one can't agree that every niche option be visible to every user, thus poluting the menus even further.
I've no use for either of these settings myself; RetroArch will eventually become unbearable to configure from scratch if every niche/personal taste setting gets added to the menu without a way to filter them out.

**Comment by tr12345** [comment]
That's a fair point I've updated both settings to have the advanced flag set.

**Comment by sonninnos** [comment]
The "Menu Controls" menu is rather short already so not sure if that advanced flag is really helpful..

**Comment by Cthulhu-throwaway** [comment]
> The "Menu Controls" menu is rather short already so not sure if that advanced flag is really helpful..

It's still another niche option one has to go through when configuring RetroArch from scratch (assuming one does it properly by going through every submenu that affects his installation).

There are already quite a few of those not flagged advanced but where the majority of users have no need to ever change them.

Other than what has already been said (and fixed), PR looks good now, imo.

**Comment by tr12345** [comment]
> The "Menu Controls" menu is rather short already so not sure if that advanced flag is really helpful..

I could go either way on this but do think it would be nice to be able to hide these settings with the advance flag.  I think for half the users they are not going to care about this setting and the default value of off is fine.  For the other half that want to use the setting they would probably turn it on once when first configuring Retroarch and then never need to touch it again.

**Comment by sonninnos** [comment]
Aye, LGTM.

**Review by sonninnos — APPROVED** [review]

**Comment by sonninnos** [comment]
Any reason why this hasn't been merged yet?

**Comment by hizzlekizzle** [comment]
w00t, this will be a popular feature for the next release.

**Pull request merged**